modules/ROOT/pages/server_release_notes.adoc

```diff
@@ -35,6 +36,25 @@ admin_manual:release_notes.adoc
 * xref:changes-in-8-0[Changes in 8.0]
 * xref:changes-in-7-0[Changes in 7.0]
 
+== Changes in 10.9.1
+
+ownCloud Server 10.9.1 is a bug fix and maintenance release.
+You can read {owncloud-server-changelog-url}[the full ownCloud Server changelog] for further details on what has changed.
+
+=== Solved known issues
```

Comment:
There are some fixes to the new version-author feature that was released with 10.9.0.

Should that be mentioned somewhere in these release notes?